lineman/app/components/lmo_href/lmo_href_for.coffee

```diff
@@ -1,11 +1,11 @@
-angular.module('loomioApp').directive 'lmoHrefFor', (LmoUrlService, LmoRedirectService) ->
+angular.module('loomioApp').directive 'lmoHrefFor', (LmoUrlService) ->
   restrict: 'A'
   scope:
     model: '=lmoHrefFor'
     action: '@lmoHrefAction'
   link: (scope, elem, attrs) ->
-    elem.attr 'href', ''
+    elem.attr 'href', LmoUrlService.route
+      model: scope.model
+      action: scope.action
     elem.bind 'click', ($event) ->
```

Comment:
The jquery ripout appears to have broken a bunch of the lineman specs, and
the e2ez don't really appear to be running either (only 3 assertions in 67
tests)

On Thu, Aug 27, 2015 at 6:14 PM, James Kiesel <james.kiesel@gmail.com>
wrote:

> Nope.
>
> On Thu, Aug 27, 2015 at 6:13 PM, Robert Guthrie <notifications@github.com>
> wrote:
>
>> In lineman/app/components/lmo_href/lmo_href_for.coffee
>> <https://github.com/loomio/loomio/pull/2443#discussion_r38152831>:
>>
>> >    restrict: 'A'
>> >    scope:
>> >      model: '=lmoHrefFor'
>> >      action: '@lmoHrefAction'
>> >    link: (scope, elem, attrs) ->
>> > -    elem.attr 'href', ''
>> > +    elem.attr 'href', LmoUrlService.route
>> > +      model: scope.model
>> > +      action: scope.action
>> >      elem.bind 'click', ($event) ->
>>
>> no angular new router dep right?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/loomio/loomio/pull/2443/files#r38152831>.
>>
>
>